lib/errors.js

```diff
@@ -91,7 +96,11 @@ class RemoteError extends Error {
   //   array - array in the form of [code, description]
   //
   static fromJstpArray(array) {
-    return new RemoteError(array[0], array[1]);
+    return new RemoteError(
+      array[0],
+      array[1],
+      array.length > 2 ? array.slice(2) : null
```

Comment:
Perhaps we can just pass an array as a third argument to avoid destructuring in `toJSTP` (and simplify this to `array[2]`)?

Reply:
I don't think that it will be better from the API user's point of view. It'll become possible to transmit an empty array having no information in it in place of the `info` over the network. And having the nested array in the message makes it somewhat harder to read for a user in my opinion.
IIUC, you don't insist on this change since you've approved the PR, so I am going to proceed with landing this PR.